server/db/add_hubs.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+original_dir=$(pwd)
+cd $(dirname "$0")
+
+if [[ $EUID > 0 ]]; then
+    echo "This script needs sudo privileges to run correctly."
+    cd $original_dir
+    exit 1
+elif [[ ! -n "$2" ]]; then
+    echo ""
+    echo "Usage: $0 path_to_.env_file path_to_hubs_csv"
+    echo "" 
+    echo "Example: $0 ./../.env ./brave_hubs.csv.example"
+    echo ""
+    echo "The hubs CSV file"
+    echo "MUST have the header 'system_id, system_name, heartbeat_alert_recipients'"
+    echo "MUST use Unix line endings (LF)"
```

Comment:
You have a copy of this file on `chatbot-dev` that is slightly different for this help text:
```
    echo "The registry CSV file"
    echo "MUST have the header 'system_id, system_name, heartbeat_alert_recipients'"
    echo "MUST use Unix line endings (LF), or else the phone numbers will have '\r' at the end"
    echo "MUST end with a newline, or else the last button will be silently ignored"
```

I've reverted it on `chatbot-dev`, but have recorded it here in case you wanted to include it in another commit.

Reply:
the version you're sharing here @trisapeace looks like it's just a more out-of-date version, ie. closer to the original `add_buttons_to_installation.sh` script. however, i did notice a few issues with this text while preparing to run the script during the leadup to the princess rooms installation. these issues are documented in https://app.clickup.com/t/faxfbg

Reply:
That makes sense. @mariocimet , can we delete the version that is chillin' on `chatbot-dev`?